### PR TITLE
DAT-3843 change Special:PortableInfoboxBuilder to Special:InfoboxBuilder

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/DAT-3843

Change will be reflected only in URL.
Reason: changing also extension name in MW app from PortableInfoboxBuilder to InfoboxBuilder would require changes in the existing InfoboxBuilder extension - an extension allowing to build infoboxes using Lua.

### CODE REVIEW

**Comment by sebastianmarzjan** [comment]
LG

**Comment by sebastianmarzjan** [comment]
LGTM

File: extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilder.i18n.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 $messages['en'] = array(
 	'portableinfoboxbuilder' => 'Infobox Builder',
```
Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilder.i18n.php]: you need to change this one to 'infoboxbuilder' for the page to be displayed correctly on Special:SpecialPages

**Comment by dianafa** [comment]
Thanks, SpecialPages looks good now.

**Pull request merged**